### PR TITLE
remove unnecessary nullable flag (#1)

### PR DESCRIPTION
**Pull request by ivyhjk**

### CODE REVIEW

File: tests/UntipedSampleTest.hh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?hh // strict
+
+use HH\Lib\Str;
```
Comment (usox) [inline tests/UntipedSampleTest.hh]: Please remove unused imports.

File: tests/UntipedSampleTest.hh [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+use function Facebook\FBExpect\expect;
+use function Usox\HackMock\{mock, prospect, tear_down};
+use Usox\HackMock\Exception\{MissingMethodCallException, UnexpectedMethodCallException};
+
+class UntipedSampleTest extends \Usox\HackMock\HackMock {
```
Comment (usox) [inline tests/UntipedSampleTest.hh]: I guess you mean `Untyped`

**Review by usox — CHANGES_REQUESTED** [review]

**Review by ivyhjk — COMMENTED** [review]
Done!

**Comment by usox** [comment]
I will perform some further tests tomorrow on some other code at work. If it works out, this will be released within a patch level release this week.

**Comment by ivyhjk** [comment]
Thanks!

**Comment by usox** [comment]
Excellent github, rebase and merge doesn't work all too good... Your Code has been merged, but github had some internal issues while rebasing/merging it

**Pull request merged**